### PR TITLE
Update SPEC to use mixed as default

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

File: resources/ottawa.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+,win_x86,win_x86-64_nocmprssptrs,win_x86-64_cmprssptrs,win_x86-64,linux_x86,linux_x86-64_nocmprssptrs,linux_x86-64_cmprssptrs,linux_x86-64,linux_ppc,linux_ppc-64_nocmprssptrs,linux_ppc-64_cmprssptrs,linux_ppc-64_nocmprssptrs_le,linux_ppc-64_cmprssptrs_le,linux_ppc-64_le,linux_390,linux_390-64_nocmprssptrs,linux_390-64_cmprssptrs,linux_390-64,aix_ppc,aix_ppc-64_nocmprssptrs,aix_ppc-64_cmprssptrs,aix_ppc-64,zos_390,zos_390-64_nocmprssptrs,zos_390-64_cmprssptrs,linux_arm,osx_x86-64_nocmprssptrs,osx_x86-64_cmprssptrs,osx_x86-64,bsd_x86-64,sunos_sparcv9-64,linux_aarch64_nocmprssptrs,linux_aarch64_cmprssptrs,linux_aarch64,linux_riscv64_nocmprssptrs,linux_riscv64_cmprssptrs
```
Comment (renfeiw) [inline resources/ottawa.csv:2]: @smlambert I assume that bsd_x86-64 and sunos_sparcv9-64 are all mixed, correct?

Comment (smlambert) [reply resources/ottawa.csv:2]: yes, assume those are mixed, thanks 👍

**Comment by renfeiw** [comment]
Tested internally on windows, linux and zos, validated the change by comparing the test output with that from master repo: Grinder_TKG/1268/, Grinder_TKG/1267/, Grinder_TKG/1265/, Grinder_TKG/1257/

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
LGTM, based on the discussion on Tuesday, is it expected to double the testing for hotspot, or is it understood why the Grinder run did not ?

**Comment by renfeiw** [comment]
> LGTM, based on the discussion on Tuesday, is it expected to double the testing for hotspot, or is it understood why the Grinder run did not ?

It was a mistake on the grinder parameter. I did another group of tests, and the executed tests number is doubled.

**Pull request merged**